### ISSUE TITLE
Error user

### ISSUE THREAD

**Issue by melchionnaeu** [instabot-py/instabot.py#1097]
> Fill in all requirements *before* submitting an issue.
> Do not delete any of the template, fill all of it in; even if you think it doesn't apply to your issue.
> If you fail to follow these simple instructions, we will close the ticket.

- [x] This is a checked box. **Do not leave spaces around the `x`!**

- [ ] I have searched for other issues with the same problem.

## Hosting
- [ ] Are you hosting the bot on a VPS or other Cloud hosting?
- [x] If so; have you tried running the code on your local machine?

#### Select one:
- [ ] Bug
- [ ] Feature Request
- [x] Technical Help
- [ ] Other

### Operating System? (include version)
MAC / Windows / Linux / Android

## Python Version?
2.7 and 3

## Description

Hi, when I try to launch `python example.py` I receive this error:
```
Traceback (most recent call last):
  File "example.py", line 52, in <module>
    unfollow_whitelist=['example_user_1', 'example_user_2'])
  File "/var/www/html/instagrambot/instabot.py/src/instabot.py", line 231, in __init__
    self.login()
  File "/var/www/html/instagrambot/instabot.py/src/instabot.py", line 300, in login
    self.user_id = ui.get_user_id_by_login(self.user_login)
  File "/var/www/html/instagrambot/instabot.py/src/userinfo.py", line 42, in get_user_id_by_login
    id_user = all_data['user']['id']
KeyError: 'user'
```

Why?

```
Include the contents of the error.log here (if applicable).
```

**Comment by NitroOxyde** [comment]
Hi, I have the same exact problem. Everything was working fine until a few days ago.

**Comment by melchionnaeu** [comment]
@NitroOxyde Now work fine with this fix [5d08de2 ](https://github.com/instabot-py/instabot.py/commit/38c46365407781e6be745dec83a4781ba0763cdb)
